src/main/kotlin/graphics/scenery/volumes/Volume.kt

```diff
@@ -183,8 +183,10 @@ open class Volume(
             }
         }
         set(value) {
+            viewerState.numTimepoints = timepointCount
```

Comment:
Shouldn't that happen at the point where the amount of timepoints is manipulated?

Reply:
Good point, I shall change that 👍